pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -82,10 +82,14 @@ class Policy(base.BasePolicy):
         queue (~queue.Queue): (Optional.) A Queue instance, appropriate
             for crossing the concurrency boundary implemented by
             ``executor``.
+        ack_on_receipt (boolean): 
```

Comment:
I do not think this knob should exist.

If this knob is to exist, it should be spelled `modack_on_receipt`. Ack and modack are _very_ different things.